caom2utils/caom2utils/tests/test_wcsvalidator.py

```diff
@@ -96,22 +96,22 @@ def test_bad_temporalwcs(self):
         bad_temporal_wcs = TimeTestUtil.bad_ctype_wcs()
         with pytest.raises(InvalidWCSError) as ex:
             wcsvalidator._validate_temporal_wcs(bad_temporal_wcs)
-        assert('unexpected TIMESYS, CTYPE' in str(ex))
+        assert 'unexpected TIMESYS, CTYPE' in repr(ex.value)
```

Comment:
`str(ex)` not working? That worries me as I've used a lot before...

Reply:
This is the installation:
platform linux -- Python 3.6.8, pytest-5.0.0, py-1.8.0, pluggy-0.12.0

And this is the error message:
```
E       AssertionError: assert 'unexpected TIMESYS, CTYPE' in '<ExceptionInfo InvalidWCSError tblen=2>'
E        +  where '<ExceptionInfo InvalidWCSError tblen=2>' = str(<ExceptionInfo InvalidWCSError tblen=2>)


```